embedded/appendable/multiapp/multi_app_test.go

```diff
@@ -234,6 +234,10 @@ func TestMultiAppEdgeCases(t *testing.T) {
 	_, err = a.ReadAt([]byte{}, 0)
 	require.Equal(t, ErrIllegalArguments, err)
 
+	err = a.Copy("multi_app_test.go")
+	require.Error(t, err)
+	require.Contains(t, err.Error(), "not a directory")
```

Comment:
comparison might compare against declared error `ErrorPathIsNotADirectory`

Reply:
The error is not generated in our side, but raised from the failed `io.Copy` operation. I could compare with `fs.PathError`, but the fs library is new in Go 1.16 and earlier version may raise something different.